indico/htdocs/sass/modules/_papers.scss

```diff
@@ -39,3 +39,92 @@
         }
     }
 }
+
+.display-contribution-list {
+    @extend %font-family-modern-body;
+
+    .contribution-row {
+        @include border-left($width: 3px);
+        padding-left: 10px;
+        margin-bottom: 1em;
+
+        &:hover {
+            border-color: $blue;
+        }
+
+        .row-title {
+            @include border-bottom();
+            font-size: 14pt;
+        }
+
+        .submitter {
+            color : $light-black;
+            margin-top: 7px;
+        }
+
+        .label {
+            border: 0;
+            margin-top: 7px;
+            display: inline-block;
+            padding: 5px 7px;
+
+            &.submitted {
+                background-color: $blue;
+            }
+
+            &.accepted {
+                background-color: $green;
+            }
+
+            &.rejected {
+                background-color: $red;
+            }
+
+            &.to_be_corrected {
+                background-color: $yellow;
+            }
```

Comment:
I suppose the comments I left in your other PR a few seconds ago, will be applied here as well :wink: